File: TaleEngine/TaleEngine/ClientApp/src/app/models/activity-dto.ts
export class ActivityDto {
    id: number;

    title: string;
    description: string;
    places: number;
    image: string;

    timeSlotId: number;

    activityStart: string;
    activityEnd: string;

    typeId: number;
    statusId: number;


    constructor(id: number, title: string, description: string, places: number, image: string,
        typeId: number, timeSlotId: number, activityStart: string, activityEnd: string,
        statusId: number) {
        this.id = id;

        this.title = title;
        this.description = description;
        this.places = places;

        this.activityStart = activityStart;
        this.activityEnd = activityEnd;

        this.typeId = typeId;
        this.timeSlotId = timeSlotId
        this.statusId = statusId;
    }
}

File: TaleEngine/TaleEngine.Bussiness.Contracts/Dtos/ActivityDto.cs
using System;

namespace TaleEngine.Bussiness.Contracts.Dtos
{
    public class ActivityDto
    {
        public int Id { get; set; }

        public string Title { get; set; }
        public string Description { get; set; }
        public int Places { get; set; }
        public string Image { get; set; }

        public int TimeSlotId { get; set; }

        public string ActivityStart { get; set; }
        public string ActivityEnd { get; set; }

        public int TypeId { get; set; }
        public int StatusId { get; set; }

    }
}
